src/layouts/components/Editable/Editable.tsx

```diff
@@ -217,7 +222,7 @@ const EditableAccordionItem = ({
           {/* NOTE: Check with design on styling.
         See if entire section is button (ie, whole component hover styling)
       */}
-          <AccordionButton px="1.5rem" py="3rem">
+          <AccordionButton px="1.5rem" py="2.125rem">
```

Comment:
assumed to be cleared with design

Reply:
Yes, the accordion was too big originally.